es/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,566 @@
 
-&#10230;
+&#10230; Observación: los métodos de mínimos cuadrados ordinarios y regresión logística son casos particulares de los modelos lineales generalizados.
 
 <br>
 
 **40. Support Vector Machines**
 
-&#10230;
+&#10230; Máquinas de vectores de soportes
 
 <br>
 
 **41: The goal of support vector machines is to find the line that maximizes the minimum distance to the line.**
 
-&#10230;
+&#10230; El objetivo de las máquinas de vectores de soportes (en inglés, Support Vector Machines) es hallar la línea que maximiza la mínima disancia a la línea.
 
 <br>
 
 **42: Optimal margin classifier ― The optimal margin classifier h is such that:**
 
-&#10230;
+&#10230; Clasificador de margen óptio - El clasificador de márgen óptimo h se define de la siguiente manera:
 
 <br>
 
 **43: where (w,b)∈Rn×R is the solution of the following optimization problem:**
 
-&#10230;
+&#10230; donde (w,b)∈Rn×R es la solución del siguiente problema de optimización:
 
 <br>
 
 **44. such that**
 
-&#10230;
+&#10230; tal que
 
 <br>
 
 **45. support vectors**
 
-&#10230;
+&#10230; vectores de soporte
 
 <br>
 
 **46. Remark: the line is defined as wTx−b=0.**
 
-&#10230;
+&#10230; Observación: la línea se define como wTx−b=0.
 
 <br>
 
 **47. Hinge loss ― The hinge loss is used in the setting of SVMs and is defined as follows:**
 
-&#10230;
+&#10230; Función de pérdida de tipo bisagra - La función de pérdida de tipo bisagra es utilizada en la configuración de SVMs y se define de la siguiente manera:
 
 <br>
 
 **48. Kernel ― Given a feature mapping ϕ, we define the kernel K to be defined as:**
 
-&#10230;
+&#10230; Núcleo - Dado un mapeo de características ϕ, se define el núcleo K como:
 
 <br>
 
 **49. In practice, the kernel K defined by K(x,z)=exp(−||x−z||22σ2) is called the Gaussian kernel and is commonly used.**
 
-&#10230;
+&#10230; En la práctica, el núcleo K definido por K(x,z)=exp(−||x−z||22σ2) es conocido como núcleo Gaussiano y es comunmente utilizado.
 
 <br>
 
 **50. [Non-linear separability, Use of a kernel mapping, Decision boundary in the original space]**
 
-&#10230;
+&#10230; [Separabilidad no lineal, Uso de un mapeo de núcleo, Límite de decisión en el espacio original]
 
 <br>
 
 **51. Remark: we say that we use the "kernel trick" to compute the cost function using the kernel because we actually don't need to know the explicit mapping ϕ, which is often very complicated. Instead, only the values K(x,z) are needed.**
 
-&#10230;
+&#10230; Observación: decimos que utilizamos el "truco del núcleo" (en inglés, kernel trick) para calcular la función de costo porque en realidad no necesitamos saber explícitamente el mapeo ϕ que generalmente es muy complicado. En cambio, solo se necesitan los valores K(x,z).
 
 <br>
 
 **52. Lagrangian ― We define the Lagrangian L(w,b) as follows:**
 
-&#10230;
+&#10230; Lagrangiana - Se define la Lagrangiana L(w,b) de la siguiente manera:
```

Comment:
I think the correct is "el lagrangiano", I've never read "la Lagrangiana" term referring to an equation